src/nvlink/wrapper.py

```
@@ -672,8 +684,15 @@ def nv_read(self) -> Tuple[int, Optional[bytes], Optional[str]]:
                 # Note: Debug log removed - this is very frequent during data fetching
                 return result, None, None
 
+            elif result in (-3, -203, -402, -403, -502, -503):
+                # Recoverable errors - caller should delete file and continue
+                # -3: ダウンロード中（該当ファイルがまだサーバーからDLされていない）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)